cloudflare/resource_cloudflare_spectrum_application_test.go

```diff
@@ -12,6 +13,50 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+func init() {
+	resource.AddTestSweepers("cloudflare_spectrum_applications", &resource.Sweeper{
+		Name: "cloudflare_spectrum_applications",
+		F:    testSweepCloudflareSpectrumApplications,
+	})
+}
+
+func testSweepCloudflareSpectrumApplications(r string) error {
+	client, clientErr := sharedClient()
+	if clientErr != nil {
+		log.Printf("[ERROR] Failed to create Cloudflare client: %s", clientErr)
+	}
+
+	zone := os.Getenv("CLOUDFLARE_DOMAIN")
+	zones, zoneErr := client.ListZones(zone)
+
+	if zoneErr != nil {
+		log.Printf("[ERROR] Failed to fetch Cloudflare zones: %s", zoneErr)
+	}
+
+	for _, zone := range zones {
```

Comment:
Realistically, I could just pull the first entry as we are only using a single domain however I think this is fine for the testing accounts.